integration-tests/tests/src/utils/ExtendedAppConfigBuilder.ts

```diff
@@ -0,0 +1,231 @@
+////////////////////////////////////////////////////////////////////////////
+//
+// Copyright 2023 Realm Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+////////////////////////////////////////////////////////////////////////////
+import {
+  AppConfigBuilder,
+  PartitionConfig,
+  CustomTokenAuthMetadataField,
+  EmailPasswordAuthConfig,
+  SyncConfig,
+} from "@realm/app-importer";
+
+import { randomDatabaseName } from "./generators";
+
+const { mongodbClusterName, mongodbServiceType = mongodbClusterName ? "mongodb-atlas" : "mongodb" } = environment;
```

Comment:
Setting the `mongodbClusterName` on the Mocha Remote context will default the service type of mongodb services to `mongodb-atlas`, since there's no reason to provide the cluster name if we're not on Atlas.

Reply:
```suggestion
// Setting the mongodbClusterName on the Mocha Remote context will default the service type of mongodb services to "mongodb-atlas"
// This makes sense because there's no reason to provide the cluster name if we're not on Atlas.
const { mongodbClusterName, mongodbServiceType = mongodbClusterName ? "mongodb-atlas" : "mongodb" } = environment;
```